src/tests/suite/daos_obj.c

```diff
@@ -706,6 +706,13 @@ io_overwrite_large(void **state, daos_obj_id_t oid)
 	/* Disabled Pool Aggrgation */
 	rc = set_pool_reclaim_strategy(state, aggr_disabled);
 	assert_int_equal(rc, 0);
+	/**
+	 * set_pool_reclaim_strategy() to disable aggregation
+	 * assumes all aggregation ULTs on all servers taking
+	 * effect immediately, this may not be the case.
+	 * Adding delay so that ULTs finish the round of aggregation.
+	 */
+	sleep(10);
```

Comment:
So there is no way to validate whether aggregation is disabled on all servers (no APIs)? I feel sleep may not be the correct method. Anyway, I will let you decide...

Reply:
Yes, I don't think so. I was hoping similar but not sure if that is possible. Assume that we have a large number of systems and a given point in time to get the correct Aggregation status. The system is idle and based on policy it may or may not trigger and the user does not have to know. From a testing perspective having some delay to settle ULT will be ok.